releasenotes/notes/add-partition-tag-b423877806ab7271.yaml

```
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    AWS: adds ``aws.partition`` tag onto AWS traces based on the region.
```

nit to clarify the integrations we are adding it to:
```suggestion
    AWS: adds ``aws.partition`` tag onto AWS traces based on the region for the boto, botocore, and aiobotocore integrations.
```